plugins/view-resources/src/components/filter/FilterTypePopup.svelte

```diff
@@ -211,15 +211,21 @@
       }
     }
   }
+
+  const elements: HTMLElement[] = []
 </script>
 
 <div class="selectPopup" use:resizeObserver={() => dispatch('changeContent')}>
   <Scroller>
     {#each getTypes(_class) as type, i}
       {#if filter === undefined && type.component === view.component.ObjectFilter && hasNested(type)}
         <Submenu
+          bind:element={elements[i]}
           on:keydown={(event) => keyDown(event, i)}
-          on:click={(event) => {
+          on:mouseover={() => {
+            elements[i].focus()
```

Comment:
```suggestion
            elements[i]?.focus()
```
I suppose it could be undefined in some rare cases.